packages/web/components/cards/pool-card.tsx

```diff
@@ -32,51 +32,52 @@ export const PoolCard: FunctionComponent<
   const t = useTranslation();
 
   return (
-    <Link href={`/pool/${poolId}`} passHref prefetch={false}>
-      <a
-        className={classNames(
-          "rounded-4xl p-[2px] text-left hover:bg-wosmongton-200",
-          {
-            "bg-osmoverse-800": !isSuperfluid,
-            "bg-superfluid hover:bg-none": isSuperfluid,
-          }
-        )}
-        onClick={() => {
-          onClick?.();
-        }}
-      >
-        <div className="flex h-full w-full cursor-pointer flex-col place-content-between gap-14 rounded-[27px] bg-osmoverse-800 px-9 py-7 transition-colors hover:bg-osmoverse-700">
-          <div className="flex place-content-between items-center">
-            <PoolAssetsIcon assets={poolAssets} />
-            <div className="ml-5 flex flex-col">
-              <PoolAssetsName
-                size="md"
-                assetDenoms={poolAssets.map((asset) => asset.coinDenom)}
-              />
-              <div className="subtitle1 flex items-center gap-1 text-white-mid">
-                {t("pools.poolId", { id: poolId })}
-                {isSupercharged && (
-                  <Icon id="lightning-small" height={16} width={16} />
-                )}
-              </div>
+    <Link
+      href={`/pool/${poolId}`}
+      passHref
+      prefetch={false}
```

Comment:
Same question. Why is prefetch false?

Reply:
No performance regression in particular, I entered it while I was fixing a re-rendering bug on the pools page, we can reset it to true. Even considering when pools appear with pagination, that's an acceptable amount of prefetches

Reply:
Same reason as above

Reply:
I enabled prefetch here